Libraries/Components/Pressable/Pressable.js

```diff
@@ -164,6 +182,20 @@ function Pressable(props: Props, forwardedRef): React.Node {
   const viewRef = useRef<React.ElementRef<typeof View> | null>(null);
   useImperativeHandle(forwardedRef, () => viewRef.current);
 
+  const _onBlur = (event: BlurEvent) => {
+    TextInputState.blurInput(viewRef.current);
```

Comment:
I don't think we need this because `TextInput` component handles this, and we probably don't want `TextInputState.currentlyFocusedField()` to return an arbitrary view.

Reply:
TextInputState holds a reference to an arbitrary component and doesn't have that much to do with TextInput actually - it's only been named that because for now in RN you can only call ref.focus() on TextInput - on Windows you can programmatically focus any View now (something we will want to implement here at some point, too). This should be refactored in the future - talked about this with Eli before: https://github.com/microsoft/react-native-windows/pull/6220#issuecomment-705955870

So this is actually necessary for keeping track of the currently focused element in a few scenarios:
if you call ref.focus(), and focus moves to the Pressable, then you use Tab to move it to another element, and then you click somewhere else on the screen, then the scrollResponder will only call TextInputState.blurInput() passing an element that no longer has focus, because it was moved with the keyboard or programmatically by calling ref.focus() on another element in whatever way. 

We will also want to implement isFocused() on View/Pressable very soon on Windows and also upstream it - this also requires that we keep track of the currently focused element.

Reply:
Got it. If that's the cast, then we'll want move this logic down into the `View` component.

Reply:
Moved the _onBlur/_onFocus callbacks down to View from Pressable. Also, set up an Ubuntu VM, and can build master now.